internal/k8s/k8s.go

```diff
@@ -603,7 +604,11 @@ func serviceNameForSlice(endpointSlice *discovery.EndpointSlice) (string, error)
 }
 
 // UseEndpointSlices detect if Endpoints Slices are enabled in the cluster.
-func UseEndpointSlices(kubeClient kubernetes.Interface) bool {
+func UseEndpointSlices(kubeClient kubernetes.Interface, cfg *Config) bool {
+	if cfg.DisableSlices {
```

Comment:
Can you add a comment about the particular scenario (epslices enabled but disabled in the proxy) that we are covering with this parameter?

Reply:
Done